class/Contact.class.php

```diff
@@ -2604,6 +2604,30 @@ function getContactPictureDetails($iduser=''){
     }
   }
   
+  /**
+    * Function to get task completed description for the contact 
+    * param int contact primary  key value.
+    *
+   */ 
+  function getTaskCompletedDetails($idcontact){
+      $q = new sqlQuery($this->getDbCon());
+      $sql = "select task_description,name from task t inner join project p on t.iduser = p.iduser inner join user u on t.iduser=u.iduser inner join contact c on c.iduser=u.iduser where c.idcontact='".$idcontact."' and t.status= 'closed' LIMIT 10";
```

Comment:
Not sure you need to link the project here as its only contact related tasks. {#16725#}